backend/src/main/java/ai/verta/modeldb/dataset/DatasetDAORdbImpl.java

```diff
@@ -635,7 +635,7 @@ public Dataset updateDatasetName(String datasetId, String datasetName)
       datasetObj.setName(datasetName);
       datasetObj.setTime_updated(Calendar.getInstance().getTimeInMillis());
       session.update(datasetObj);
-      transaction.commit();
+      //transaction.commit();
```

Comment:
this transaction is required because we are doing `Update` operation here without transaction update is not work.